app/src/docs/v1.api-spec.yaml

```diff
@@ -2245,6 +2259,13 @@ components:
                 An entity tag (ETag) is an opaque identifier assigned by a web
                 server to a specific version of a resource found at a URL
               example: '"9d1aaa54b84e1d6ccc6e0477c5717fe3"'
+            lastUpdatedVersion:
+              type: string
+              format: date-time
+              description: >-
+                Time when the version was last brought into COMS via sync
```

Comment:
the `lastModifiedDate` is an attribute on the S3 object/version itself. whenever you create a version/object in s3 or maybe update the tags, the lastModified date is changed.
We plan to start recording this in our db version table as well.
it will be usually be populated when someone uploads a file.

For the description I would put something like:
`The object creation date or the last modified date, whichever is the latest.`